### ISSUE TITLE
Error deactivating package

### ISSUE THREAD

**Issue by idleberg** [suda/tool-bar#328]
I noticed a minor issue, the package doesn't properly deactivate when Atom is closed/reloaded.

```
events.js:200 Error deactivating package 'tool-bar' TypeError: Cannot read property 'destroy' of null
    at Object.exports.deactivate (/Users/jan/.atom/packages/tool-bar/dist/tool-bar.js:1:9575)
    at Package.deactivate (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3197328)
    at PackageManager.deactivatePackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:384660)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:384402
    at Array.map (<anonymous>)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:384390
    at Config.transactAsync (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:332389)
    at PackageManager.deactivatePackages (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:384335)
    at AtomEnvironment.prepareToUnloadEditorWindow (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:718779)
    at EventEmitter.t (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:734406)
```

Atom    : 1.52.0
Electron: 6.1.12
Chrome  : 76.0.3809.146
Node    : 12.4.0
tool-bar: 1.4.1

**Comment by ericcornelissen** [comment]
I'm pretty sure this issue was fixed by #326, which still needs to be released @suda

**Comment by suda** [comment]
Oh sorry about that, just released 1.4.2 with this fix :+1:
